ironfish/src/rpc/routes/wallet/getTransactions.ts

```diff
@@ -33,6 +39,7 @@ export type GetAccountTransactionsResponse = {
   expiration: number
   timestamp: number
   assetBalanceDeltas: Array<{ assetId: string; assetName: string; delta: string }>
+  notes?: RpcAccountDecryptedNote[]
```

Comment:
I am partial to `notes: RpcAccountDecryptedNote[] | undefined` since it is more explicit, not sure if this is the pattern we use though

Reply:
I suspect that we're not very consistent about using one vs the other. I'll add a task to the RPC uniformity project to address this and make it consistent!